Controller/RedirectRouteImportController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RedirectBundle\Controller;
+
+use Sulu\Bundle\RedirectBundle\Import\Converter\ConverterNotFoundException;
+use Sulu\Bundle\RedirectBundle\Import\FileImportInterface;
+use Sulu\Bundle\RedirectBundle\Import\Item;
+use Sulu\Bundle\RedirectBundle\Import\Reader\ReaderNotFoundException;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides API to upload importable files.
+ */
+class RedirectRouteImportController
```

Comment:
Permission check missing user need `edit and create` permissions for import? /cc @danrot